sirepo/mpi.py

```diff
@@ -8,6 +8,7 @@
 from pykern import pkio
 from pykern import pksubprocess
 from pykern.pkdebug import pkdc, pkdexc, pkdp, pkdlog
+from sirepo.template import template_common
```

Comment:
I don't think this is a good idea. There may be circular import problems. You could put it in sirepo.const, and this would avoid that issue.

Reply:
👍